staging/cse/windows/debug/collect-windows-logs.ps1

```diff
@@ -176,53 +177,63 @@ else {
   Write-Host "Get-PSDrive command not available"
 }
 
+Write-Log "Collecting available memory"
```

Comment:
Should we use ```Write-Host``` here? I checked the collected node logs and found warning about ```Write-Log``` in collect-windows-logs-output.txt

Reply:
Good catch. Fixed it.